olp-cpp-sdk-dataservice-read/src/MultiRequestContext.h

```diff
@@ -38,7 +38,7 @@ namespace dataservice {
 namespace read {
 
 namespace {
-constexpr auto kLogTag = "MultiRequestContext";
+constexpr auto kMultiRequestContextLogTag = "MultiRequestContext";
```

Comment:
in some places you rename kLogTag but not in the other (e.g. in PrefetchTilesProvider)

Reply:
implementation files work perfectly in  separate files while MultiRequestContext is an in-header implementation which conflicts with other declarations of kLogTag